src/interactions/slashCommands/SlashCommandBuilder.ts

```diff
@@ -44,16 +44,12 @@ export class SlashCommandBuilder {
 		input:
 			| SlashCommandSubcommandGroupBuilder
 			| ((subcommandGroup: SlashCommandSubcommandGroupBuilder) => SlashCommandSubcommandGroupBuilder),
-	): SlashCommandSubcommandGroupsOnlyBuilder {
+	): SlashCommandSubcommandsAndSubcommandGroupsOnlyBuilder {
```

Comment:
`SlashCommandsBuilderWithoutOptions` maybe? @discordjs/the-big-3

Reply:
`SlashCommandsBuilder` and `SlashCommandsBuilderWithOptions` maybe?

Reply:
Former is the general, non strict type. Can't use it for this..

Reply:
Then yes, `SlashCommandsBuilderWithoutOptions` sounds good

Reply:
Are subcommands not technically options as well?

Reply:
`SlashCommandsSubcommandsOnly` will be enough for now.. Since there will be a rename later